pkgs/firehose/bin/health.dart

```diff
@@ -48,17 +48,23 @@ void main(List<String> arguments) async {
     ..addFlag(
       'coverage_web',
       help: 'Whether to run web tests for coverage',
+    )
+    ..addMultiOption(
+      'flutter_packages',
```

Comment:
I see we're using a `foo_bar` naming convention here and for the other options. I believe a `foo-bar` convention is more common for args, but likely not worth changing here or being inconsistent w/ the other option names.